### PR TITLE
Allow flexible scheme in identifier URLs via identifier_director.

### LINKED ISSUE #11: dc_structure.identifier_director hardcodes the http scheme in permalinks

### PR DESCRIPTION
**Pull request by ldko**
This should fix #11 by allowing a scheme to be passed for use.

@somexpert @jason-ellis @jthomale

### CODE REVIEW

File: pyuntl/dc_structure.py [hunk 1]
```diff
@@ -248,6 +248,8 @@ def identifier_director(**kwargs):
     """Direct how to handle the identifier element."""
     ark = kwargs.get('ark', None)
     domain_name = kwargs.get('domain_name', None)
+    # Set default scheme if it is None or is not supplied.
+    scheme = kwargs.get('scheme') or 'http'
```
Comment (somexpert) [inline pyuntl/dc_structure.py]: For consistency with the other changes you've made, can we make this `scheme = kwargs.get('scheme', 'http')`?

**Comment by ldko** [comment]
It is handled that way so if the function gets called with `scheme=None` (as opposed to calling without passing `scheme` at all) it will still get 'http' value.

**Comment by somexpert** [comment]
Ah, I see what you mean now.

**Comment by somexpert** [comment]
:thumbsup:

**Comment by jthomale** [comment]
:+1:

**Pull request merged**